pyproject.toml

```diff
@@ -9,9 +9,10 @@ cmake.verbose = true
 
 [project]
 name = "green-igen"
-version = "0.2.6"
+version = "0.2.7"
 authors = [
   { name="Sergei Iskakov", email="siskakov@umich.edu" },
+  { name="Gaurav Harsha" },
```

Comment:
propose to remove Sergei's email and add yours so that comments come to you rather than Sergei.